gh-md-toc

```diff
@@ -275,6 +269,7 @@ gh_toc_app() {
 
     if [ "$1" = '--insert' ]; then
         need_replace="yes"
+        no_backup="yes"
```

Comment:
Hi @jordantrizz  I don't think this going to get you what you are expecting. Placing no_backup="yes" on line 272 will cause the "--insert" and "--no_backup" CLI options to be related and that may not be what folks want or expect. To keep them independent, you may want to remove this line and add the following:
line 233:  local no_backup="no"
And then change line 182 as follows:
from:
   if [ -z $no_backup ]; then
to:
   if [ $no_backup = "no" ], then

I think this will keep the two CLI options independent and the code will provide the expected results.

Reply:
Totally, which I had time to commit to this but I don't :(

Reply:
This has been fixed differently in #108